schemas/SendMessage.json

```diff
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "description": "Triggered when the user sends a message",
+  "properties": {
+    "eventName": {
+      "enum": ["SendMessage"]
+    },
+    "roomId": {
```

Comment:
Room Ids are PII, we won't send them as analytics data. What do you need this for?